habitat_sim/utils/datasets_download.py

```diff
@@ -101,6 +120,59 @@ def initialize_test_data_sources(data_path):
         },
     }
 
+    data_sources.update(
+        {
+            f"hm3d_{split}_{data_format}": {
+                "source": "https://api.matterport.com/resources/habitat/hm3d-{split}-{data_format}.tar{ext}".format(
```

Comment:
Wouldn't this jsut be cleaner with fstrrings.

Reply:
IMO no because of the need to add .gz to the obj+mtl tarballs

Reply:
f strings can have nested ifs. *shrugs*